### DISCUSSION TITLE
Prepare presentation for the PN SF and we will present it in coming JMAR meeting

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#28] [Comments_HIG_23_001]
Tab 10-12: did you calculate these or are they provided by the POG? What is included in the uncertainties? If you calculated them yourself, have they been shown to the POG?
General on the PNet calibration: these are scale factors for the signal (ie anything with Z->jets), how are you determining scale factors for the ttbar and WW processes?

**Comment by ram1123** [comment]
@jialin-guo1: Once you have the presentation ready, let us know.

Let's aim that you present this at Tuesday's meeting.

**Reply by jialin-guo1** [reply]
@ram1123 

https://www.icloud.com.cn/keynote/0f0tUKeTsHe4HHb9h5gS9KmJw#ParticleNetCalibration

I made a very beginning slide for you review. most of content are there. I need to adjust layout and format then

**Reply by ram1123** [reply]
@jialin-guo1 : Some quick comments on your presentation:

General comments:
- Add brief introduction of our analysis
   - Also, add the event selection and the flow chart
   - Add the control plots along with the PN score control plot showing that it has the issue and we need to calibrate
- Add the GitHub repo link for the framework that you used for the PN calibration

Comments on each slide:
- slide - 7: Zoom in the pT plot
- slide - 8: before this add the some details about the method that it is similar to the traditional tag and probe method, etc. etc.

**Comment by ram1123** [comment]
- JMAR indico: https://indico.cern.ch/event/1304758/contributions/5487549/attachments/2682024/4660227/ParticleNetCalibration.pdf
- BTV indico: https://indico.cern.ch/event/1324124/#6-review-of-particlenet-z-tagg
- Jialin webpage with SFs: https://guoj.web.cern.ch/guoj/HZZ2L2Q/UL_legacy/netcali/
